### ISSUE TITLE
Admin deposit view is broken after #740

### ISSUE THREAD

**Issue by ysv** [openware/peatio#809]
Probably problem in this line https://github.com/rubykube/peatio/commit/859031d5c74c2cb0ec7e10f6cc7344d0541c7cc0#diff-a33daedbd6f42e19fca2e90e6a85cd51L36

**Comment by yivo** [comment]
Duplicate of #795.
